src/firebase/firebase-config.js

```diff
@@ -0,0 +1,16 @@
+import firebase from 'firebase/app';
+import 'firebase/auth';
+
+const firebaseConfig = {
+  apiKey: 'AIzaSyBeupwCnetKwa7W81gIpY8WmKOVrP52cD0',
+  authDomain: 'tutorial-firebase-ea333.firebaseapp.com',
+  databaseURL: 'https://tutorial-firebase-ea333.firebaseio.com',
+  projectId: 'tutorial-firebase-ea333',
+  storageBucket: 'tutorial-firebase-ea333.appspot.com',
+  messagingSenderId: '166869096241',
+  appId: '1:166869096241:web:206af38029d559cfcd8210',
+};
+
+firebase.initializeApp(firebaseConfig);
+
+export { firebase };
```

Comment:
Remember to **never put credentials on the code**. 

Try to use Env vars instead always. Please fix this.

Reply:
This is because with this credentials, I can use them and access to your firebase data.